### ISSUE TITLE
Will pretrained models converge faster on downstream tasks than non pretrained models?

### ISSUE THREAD

**Issue by xylcbd** [keyu-tian/SparK#24]
Is there any loss curve for that? thanks

**Comment by keyu-tian** [comment]
Yes we provide some tensorboard log like in https://github.com/keyu-tian/SparK/tree/main/downstream_d2#fine-tuned-resnet-50-weights-log-files-and-performance.

I couldn't find the experiment log of non-pretrained finetuning, but I found a MoCo V2 log. Here's an AP curve of ResNet-50-SparK (blue) vs. ResNet-50-MoCoV2 (orange). It's a COCO MaskRCNN FPN 1x finetuning configuration.

![loss](https://user-images.githubusercontent.com/39692511/231512895-a5fb8f5f-8327-42ab-9cd1-dc4719f3bfdd.png)

I think MoCo V2 should converge faster than non-pretrained, so the advantage of SparK is evident.
